### ISSUE TITLE
mocking withPreview component in jest gives TypeError: Cannot assign to read only property '__esModule' of object '[object Object]'

### ISSUE THREAD

**Issue by ShreyasKraman** [birkir/gatsby-source-prismic-graphql#116]
I am trying to run test cases for components which make use of withPreview function. Therefore I am trying to mock withPreview in __mocks__/gatsby.js folder as provided by gatsby so that withPreview function works the way it should. the test environment is setup as per https://www.gatsbyjs.org/docs/unit-testing/

Error I am getting:

![image](https://user-images.githubusercontent.com/40546118/71028241-ef6bcc00-20da-11ea-8132-c8ecf217473d.png)

I tried absolute import as per https://github.com/ReactTraining/react-router/pull/6757. Does not work still getting the same error below:

![image](https://user-images.githubusercontent.com/40546118/71027592-8b94d380-20d9-11ea-9ef4-20bb448a8f43.png)

As per https://github.com/ReactTraining/react-router/issues/6649, timdorr says updating apollo-boost works. 

Currently apollo-boost version is: 

![image](https://user-images.githubusercontent.com/40546118/71028642-b3853680-20db-11ea-88d0-fe7b5d1c5007.png)

I am not sure if I can update apollo-boost in gatsby-source-prismic-graphql.

Kindly advise the way to resolve this issue.

**Comment by bostrom** [comment]
@ShreyasKraman it's been a while, but did you solve this?

**Comment by ShreyasKraman** [comment]
@bostrom No, unfortunately havent been able to solve this. Last i had chat with prismic, they told to clone this plugin repo and update the apollo library and then send a pull request. I have not really tried that approach. Are you facing this same issue?
